src/main/app/src/components/hakutulos/hakutulosKortit/KoulutusKortti.tsx

```diff
@@ -55,11 +55,11 @@ export const KoulutusKortti = ({ koulutus, isSmall }: Props) => {
       .replace(/<\/li>/gm, ',</li>')
       .replace(/\.,<\/li>/gm, '.</li>')
       .replace(/<[^>]*>/gm, '') || t('haku.ei_kuvausta');
-  const isOsaamisalaOrTutkinnonOsa = _.includes(
-    ['amm-osaamisala', 'amm-tutkinnon-osa'],
+  const isOsaamisalaOrTutkinnonOsaOrAmmuu = _.includes(
```

Comment:
Tämä ehto kannattais kääntää toisin päin ja laittaa muuttujalle nimeksi esim. `hasTutkintonimike`.

Reply:
Huomasin että KoulutusInfoGrid:ssä on hyvin samanlaista koodia. Nyt eivät ainakaan ole ihan samanlaiset. Kannattaa katsoa voisko jakaa koodia.